longevity_lwt_test.py

```diff
@@ -0,0 +1,87 @@
+import re
+
+from longevity_test import LongevityTest
+from sdcm.sct_events import LwtFailureEvent
+
+
+class LWTLongevityTest(LongevityTest):
+
+    def test_lwt_longevity(self):
+        self.test_custom_time()
+        self.validate_data()
+
+    def validate_data(self):
+        node = self.db_cluster.nodes[0]
+        with self.cql_connection_patient(node, keyspace='cqlstress_lwt_example') as session:
+            self.validate_not_updated_data(session=session)
+            self.validate_updated_data(session=session)
+
+    def validate_not_updated_data(self, session):
+        """
+        User profile 'c-s_lwt_data_not_for_update.yaml' prepares data for "blogposts" table that used in the LWT
+        longevity. This data won't be updated during the test because of values for partition keys are higher then max
```

Comment:
The explanation isn't clear enough.
I know what you are trying to do and it was hard for me to follow.

Reply:
Changed function logic. Hope it's more clear now